en/documentation/index.md

```diff
@@ -135,13 +118,8 @@ If you have questions about Ruby the
 [9]: http://www.ruby-doc.org/docs/ProgrammingRuby/
 [10]: http://pragmaticprogrammer.com/titles/ruby/index.html
 [12]: http://en.wikibooks.org/wiki/Ruby_programming_language
-[13]: http://www.ruby-doc.org/core
-[14]: https://ruby.github.io/rdoc/
-[15]: http://www.ruby-doc.org/stdlib
```

Comment:
Both `[13]` and `[15]` redirects to <https://ruby-doc.org/3.3.4/> now, so I think we don't have to remove these?

Reply:
`[34]` also links to <http://ruby-doc.org>, it needs to be https though.